Plugins/Source/KISAddonPickup.cs

```diff
@@ -873,10 +873,26 @@ public enum PickupMode {
 
   private IEnumerator WaitAndStopDrag() {
     yield return new WaitForFixedUpdate();
-    icon = null;
+    DisableIcon();
     draggedPart = null;
   }
 
+  // Sets icon, ensuring any old icon is Disposed
```

Comment:
Sorry, thought I had done.  Been fighting with monodevelop to stop screwing around with my code formatting.  I'll be more vigilant in future before submitting.